src/shared/components/Header.tsx

```diff
@@ -0,0 +1,16 @@
+import styled from "styled-components";
+import { Link } from "react-router-dom";
+
+export const Header = styled.div`
+  flex-direction: row;
+`;
+
+export const StyledLink = styled(Link)`
+  width: fit-content;
+  margin-bottom: 1em;
+  margin-right: 1em;
+  padding: 0.5em 0.75em;
+  border: 1px solid #fff;
+  border-radius: 24px;
+  font-size: 1.5em;
+`;
```

Comment:
It's probably better to make reusable components where you can, I didn't spend too long on the styling as I can see we have an internal library for many components so didn't think it was worth too much effort.

Reply:
Ooh, one other thing of note, I believe `styled-components` overrides the `className` of the components it's wrapping with an internal ID, so that makes it a little bit harder to write your own CSS classes.

There's a bit more about it here: https://styled-components.com/docs/advanced#existing-css

Reply:
Yeh I read about it as well that they are scoped to components so component styles wont be available elsewhere and needs to be done global level which is bit anti-pattern.